docker-compose-mongo.yml

```diff
@@ -7,3 +7,5 @@ services:
     # allows contact with mongo from localhost outside a container
     ports:
       - "127.0.0.1:27017:27017"
+    volumes:
+      - /mnt/data/db:/data/db
```

Comment:
please add a comment to this file that calls out that it is intended for dev use only, not prod since /mnt may disappear at any time